dotCMS/src/main/java/com/dotcms/rest/api/v1/experiment/Experiment.java

```diff
@@ -0,0 +1,95 @@
+package com.dotcms.rest.api.v1.experiment;
+
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.rules.model.Rule;
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+
+public class Experiment {
+
+    private String name;
+    private String key;
+    private ExperimentStatus status;
+    private Collection<Rule> targeting;
+    private boolean uniquePerVisitor;
+    private int lookBackWindowMinutes;
+
+    private Collection<AnalyticEvent> events;
+    private VariantsCollection variants;
+
+
+    public Experiment(final String key, final String name, final ExperimentStatus status, final boolean uniquePerVisitor, final int lookBackWindowMinutes) {
```

Comment:
to many of them, introduce a Builder would be a good idea

Reply:
It's a good candidate for a Builder for sure